ui/src/layouts/chats/presentation/chat/controls.rs

```diff
@@ -53,11 +54,18 @@ pub fn get_controls(cx: Scope<ChatProps>) -> Element {
         .unwrap_or(false);
 
     let call_pending = use_state(cx, || false);
+    let show_more = use_state(cx, || false);
     let active_call = state.read().ui.call_info.active_call();
     let call_in_progress = active_call.is_some(); // active_chat.map(|chat| chat.id) == active_call.map(|call| call.conversation_id);
 
     let show_pinned = use_state(cx, || false);
 
+    use_effect(cx, &minimal, |_| {
+        to_owned![show_more];
+        async move {
+            show_more.set(false);
+        }
```

Comment:
recommend this:

```
if show_more {
    show_more.set(false);
}
```

Reply:
marking this as resolved as flem addressed it